### PR TITLE
Feature/9 convert css to sass

### PR DESCRIPTION
**Pull request by EdgarOPG**
* Refactored Css stylesheet to Sass
* Removed script tags from template in order to be loaded from webpack.
* Refactored js files in order to have only one class on each file.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,5 +1,30 @@
-require('./src/rrDisplay');
-require('./src/Claim');
-require('./src/Root');
-require('./src/score');
-require('./src/SettleIt');
\ No newline at end of file
+//TODO Change js libs to their ts versions
+
+//Import JS
+hyperHTML = require('./HyperHTML.js');
```
Comment (rrgarciach) [inline index.js]: This is just an example of a review comment

**Comment by BentleyDavis** [comment]
On this branch I get :
```
ERROR in ./index.js
Module not found: Error: Can't resolve 'style-loader' in 'C:\Users\Bentley\Documents\GitHub\ReasonRoot'
 @ ./index.js 13:0-28
```
when I run `webpack`

**Comment by BentleyDavis** [comment]
I can go ahead and commit either or both of these if it will be fixed in a later commit.

**Comment by BentleyDavis** [comment]
These should really be a pull request to the `dev` branch. Mast is automatically deployed so it is after it has been tested.

**Comment by EdgarOPG** [comment]
Lets see,'style-loader' is new dependency that allow us to manage stylesheets through webpack, that error can be fixed running:
`npm i`

About this should be a pull request to `dev`
My fault, I should have asked before. You can reject this, and I will do the pull request to dev.

**Comment by BentleyDavis** [comment]
Yes, I forgot I need to do an npm install to get the latest code.

**Comment by rrgarciach** [comment]
I will reopen this PR and switch it to `dev` branch instead of `master`.
@EdgarOPG next time let's keep this on mind when reating PR ;)

**Pull request merged**